README.md

```diff
@@ -9,3 +9,43 @@ Status and support
 - &#x2716; work in progress
 - &#x2714; supported
 - &#x2714; ongoing development
+
+### Usage
+
+Include the CDN in the desired file.
+```html
+<script type="text/javascript" src="https://cdn.jsdelivr.net/gh/c-hive/team-contribution-calendar/dist/team-contribution-calendar.min.js">
```

Comment:
Why `jsdelivr` instead of GitHub?

Reply:
I have already mentioned some possibilites [**here**](https://gitlab.com/c-hive/c-hive.dev/issues/16#note_159564755). Otherwise, we haven't ended up with any concrete CDN provider yet. 

I'm not sure that GH provides such a feature(expect [**rawGit**](https://rawgit.com/), but this isn't GH specific).

Reply:
Doesn't this work?

https://raw.githubusercontent.com/c-hive/team-contribution-calendar/master/dist/index.min.js

Reply:
IMO it should work. Thanks.

Reply:
Unfortunately, it seems that it's not the proper way of accessing files like this via links. I tried to do the same with `github-calendar` but encountered with the errors below:

![Képernyőfotó 2019-04-19 - 16 39 14](https://user-images.githubusercontent.com/30864202/56429092-f6c7a500-62c1-11e9-9e9a-3b82d13c1102.png)

Reply:
I guess it's  jsdelivr then. 👍